lesson_06/homeworks_06.py

```diff
@@ -1,72 +1,73 @@
-# task 1
-"""  Уявіть, що інопланетянина з кольором alien_color щойно збили в грі.
-Створіть змінну під назвою alien_color і призначте їй значення 'green', 'yellow', або 'red'.
-Напишіть оператор if, щоб перевірити, чи колір прибульця 'green'.
-Якщо так, надрукуйте повідомлення про те, що гравець щойно заробив 5 балів.
-"""
+alien_color = 'green'
```

Comment:
```suggestion
alien_color = 'green'
```
а де тут виконано Напишіть оператор if, щоб перевірити, чи колір прибульця 'green'
я не бачу, незараховано.